action-scripts/brownie/scripts/pokeInjectors.py

```diff
@@ -4,8 +4,7 @@
 
 
 CHAINS_TO_KEEP = [
-    #    "zkevm-main", # currently running in defender due to rpc issues
-    "gnosis-main",
+    "zkevm-main",
```

Comment:
The poker is permissioned, so this call will fail now.  

If we want to swap keepers, we need to update the keeper on the underlying contracts and then disable oz + enable zkEVM.

As I recall we're not over our usage limit right now and/or if we are it's not by much. 

Suggest handling zkEVM as a separate ticket/work item and making sure we dot all the Is and cross all the Ts.

Reply:
moved to #1333